CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # 7.0.0, in progress
 
+## Added
+* Veneur can be set to flush steady OK values for service checks only once per configurable time interval. Veneur will still emit non-OK values and changes to OK from other values. Thanks, [antifuchs][antifuchs]!
```

Comment:
I think this warrants inclusion in the general docs/README or something. The idea that it skips steady-state OKs but emits any !OK status took me a bit to infer even though you said it here. 😄 

Also, this will inherents reset on veneur reset. This is pretty obvious if you grok the implementation but seems work documenting.

Reply:
Good idea. I should also clarify that the default is to not sample the OK states at all - that only happens if you configure an interval. Will doc-ify.

Reply:
And good point about the reset! I initially thought that it would be possible to emit events when this detects that a check had switched states (if we didn't throw away that map every n minutes), but I think the number of times we restart veneur (not rarely) means it would flood events everywhere.